lib/admin_plugins/futureitems.js

```diff
@@ -0,0 +1,151 @@
+'use strict';
+
+var futureitems = {
+  name: 'futureitems'
+  , label: 'Remove future items from mongo database'
+  , pluginType: 'admin'
+};
+
+function init() {
+  return futureitems;
+}
+
+module.exports = init;
+
+futureitems.actions = [
+    {
+        name: 'Find and remove treatments in the future'
+      , description: 'This task find and remove treatments in the future.'
+      , buttonLabel: 'Remove treatmens in the future'
+    }
+    , {
+        name: 'Find and remove entries in the future'
+      , description: 'This task find and remove CGM data in the future created by uploader with wrong date/time.'
+      , buttonLabel: 'Remove entries in the future'
+    }
+  ];
+
+futureitems.actions[0].init = function init(client, callback) {
+  var translate = client.translate;
+  var $status = $('#admin_' + futureitems.name + '_0_status');
+  
+  function showTreatments(treatments, table) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [This function's cyclomatic complexity is too high. (8)](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/995071951/issues/source?bid=2452379&fileBranchId=2865970#l32)